torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -292,7 +292,8 @@ void ProcessGroupAgent::shutdownImpl() {
 
 std::shared_ptr<FutureMessage> ProcessGroupAgent::send(
     const WorkerInfo& to,
-    Message&& message) {
+    Message&& message,
+    const float rpcTimeout) {
```

Comment:
not required: I've found it can be helpful in some cases to add the unit to the variable name in apis.
e.g. rpcTimeoutSeconds or rpcTimeoutMilliseconds

btw, if it is seconds (which is common for float), you might want to multiply by 1000 below before casting

Reply:
I originally meant for this to be in ms, although, not sure what was intended when the `float` type was suggested (cc @pritamdamania87 )

Reply:
fwiw, in many/most cases where I've seen timeouts represented as a `float`, the unit is seconds.

Whatever we got with, adding units to the var name might help self-document and eliminate confusion. I've seen a ton of seconds/msec/usec mixups...

Reply:
@jjlilley Updated to be seconds and changed the param to `rpcTimeoutSeconds`